### ISSUE TITLE
Create a basic structure for the report

### ISSUE THREAD

**Issue by chrarvi** [elias-sundqvist/DATX02-20-04#28]
Structure it in such a way that people can easily go in and write as they work

- Introduction
  - Purpose
  - Method
  - Scope
- Background
  - Music (Unsure of this one here)
  - Previous works
    - Text to Speech
    - WaveNET
    - Music Transformer
    - MuseNET

- Theory
  - Signal processing for audio analysis
    - Frequencies and overtones
    - Spectrograms
  - Audio format
    - MIDI
    - WAV files
  - Machine learning
    - What is machine learning
    - Deep neural networks
    - Loss functions
      - Squared error
      - Cross entropy
      - Wasserstein (Too many indents, ask Marwa)
    - Optimizers
  - Machine learning models
    - Feed-forward
    - Convolutional neural network
    - Generative adversarial network
    - Variational Auto encoders
    - Transformers
    - Variational auto encoders
- Implementation
- Result
- Discussion
  - Ethical and societal aspects
- Conclusion
